cmd/bundle/schema.go

```diff
@@ -16,11 +17,29 @@ func newSchemaCommand() *cobra.Command {
 	}
 
 	var openapi string
+	var outputFile string
 	var onlyDocs bool
 	cmd.Flags().StringVar(&openapi, "openapi", "", "path to a databricks openapi spec")
 	cmd.Flags().BoolVar(&onlyDocs, "only-docs", false, "only generate descriptions for the schema")
+	cmd.Flags().StringVar(&outputFile, "output-file", "", "File path to write the schema to. If not specified, the schema will be written to stdout.")
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		// If no openapi spec is provided, try to use the environment variable.
+		// This environment variable is set during CLI code generation.
+		if openapi == "" {
+			openapi = os.Getenv("DATABRICKS_OPENAPI_SPEC")
+		}
+
+		// If outputFile is provided, write to that file.
+		if outputFile != "" {
+			f, err := os.OpenFile(outputFile, os.O_WRONLY, 0644)
```

Comment:
* Need `os.O_TRUNC`?
* This should only open and truncate the file when we're ready to write.